tests/virtual_machines_test.go

```diff
@@ -298,48 +279,55 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 		Context("When trying to create a VM after all MAC addresses in range have been occupied", func() {
 			It("[test_id:2162]should return an error because no MAC address is available", func() {
-				err := initKubemacpoolParams("02:00:00:00:00:00", "02:00:00:00:00:01")
-				Expect(err).ToNot(HaveOccurred())
-
+				if shouldSkipTest() {
+					Skip("Skipping test on production platforms")
```

Comment:
I think we can call `Skip` inside `shouldSkipTest` so we don't need the if statement the function should be named then `skipTestIfNeeded` or the like, We can even call it at a `BeforeEach` so e can forget about it.

Also the Skip string is different at all the calls here.

Reply:
ok I moved it inside `skipTestIfNeeded`. we can't move it to `BeforeEach` though since we only want these 3 test to be skipped.